src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,16 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    public boolean equals(Rectangle ob) {
```

Comment:
@kenu21 did you override this method from object?

Reply:
> did you override this method from object?
From Object? Yes, I did.

Reply:
No) You didn't)

Reply:
> No) You didn't)
And now?)